core/templates/pages/admin-page/services/admin-router.service.ts

```diff
@@ -29,6 +29,13 @@ export class AdminRouterService {
   currentTabHash: string = (
     AdminPageConstants.ADMIN_TAB_URLS.ACTIVITIES);
 
+  /**
+   * Iterates through the ADMIN_TAB_URLS map and returns the
+   * tab name corresponding to the hash.
+   * @param {string} tabHash Represents hash name of a tab URL.
```

Comment:
`tabHash The string after the '#' character in the URL.`

Reply:
Done